gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
`||` is going to collapse all nonzero `sort` statuses into whatever status gets passed to the following `exit`.

This check is already assuming we have GNU sort. It's just making sure it's a version that supports `-V`. Distinguishing GNU sort from BSD support would probably be a separate check, and one that's not worth doing since we don't support any BSD-based OS'es yet.